.ci/test-project/uberenv_config.json

```diff
@@ -0,0 +1,10 @@
+{
+"package_name" : "magictestlib",
+"package_version" : "1.0.0",
+"package_source_dir" : ".",
+"spack_build_mode" : "uberenv-pkg",
+"spack_configs_path": "spack_configs",
+"spack_packages_path": "packages",
+"spack_url": "https://github.com/spack/spack",
+"spack_branch": "develop"
```

Comment:
Do we want to tie this to `spack@develop` or  lock it down to a specific commit/tag?

Reply:
I say we pin *if* we see problems.  The hope is that this is so basic that spack changes shouldn't be able to undermine it.  That might not be the case when we add things like envs, so we can reconsider later.